pages/common/ssh-keygen.md

```diff
@@ -14,7 +14,7 @@
 
 `ssh-keygen -t ed25519 -a 100`
 
-- Generate an RSA 4096 bit key with email as a comment:
+- Generate an RSA 4096-bit key with email as a comment:
```

Comment:
Not sure this is really necessary. Personally I'd prefer it with a space, not a dash.

Reply:
Oh damn, peeking around on Wikipedia it does seem space vs hyphen isn't strict.
I thought hyphen was standard.

Also, a snippet from ssh-keygen itself:
```
compact format, taking as little as one bit per certificate if they are
             numbers are 64-bit values, not including zero and may be ex‐
             file using 128-bit AES.  This file is not automatically accessed
```

We'll see what others think, but I can revert this if preferred.

Reply:
I like it with the hyphen, tbh.

Reply:
Ah, ok. If the official documentation for the command uses a hyphen, we should too in this specific instance.